docs/tutorials/tabular/tabular-quick-start.ipynb

```diff
@@ -25,13 +24,21 @@
    },
    "outputs": [],
    "source": [
-    "!python -m pip install --upgrade pip\n",
-    "!python -m pip install autogluon"
+    "import sys\n",
+    "if sys.platform == \"emscripten\":\n",
+    "    %pip install autogluon-lite-common\n",
+    "    %pip install autogluon-lite-core\n",
+    "    %pip install autogluon-lite-core\n",
+    "    %pip install autogluon-lite-tabular\n",
+    "    %pip install autogluon-lite\n",
+    "else:\n",
+    "    !python -m pip install --upgrade pip\n",
+    "    !python -m pip install autogluon"
```

Comment:
We should try as hard as possible to avoid users getting confused when looking at their very first tutorial. This code block makes installing AutoGluon feel confusing, since they won't know what "emscripten" is.

If possible, we should only have the relevant pip install commands exist/visible based on the user's env.

Reply:
Sure, i can try to fold these installation into pyodide itself. That could hide these completely. Here is pros and cons:

pros:
* less intrusive change to existing tutorial
* more tutorials can be included without intrusive modification

cons:
* extra loading time of pyodide itself (installing all autogluon packages under the scene)
* customizing pyodide can be a bit tricky

Reply:
I think the pros outweigh the cons here, at least for now